test-specifications/crypto-service/accountApproveAllowanceTransaction.md

```diff
@@ -0,0 +1,245 @@
+# AccountAllowanceApproveTransaction - Test specification
```

Comment:
Does this file need to be in this PR? I believe it's already implemented in separate PR.